src/App.tsx

```diff
@@ -163,8 +162,7 @@ const providers = [
   wrapProvider(StatsigProvider),
   wrapProvider(QueryClientProvider, { client: queryClient }),
   wrapProvider(GrazProvider),
-  wrapProvider(PrivyWagmiConnector, { wagmiChainsConfig: configureChainsConfig }),
-  wrapProvider(WagmiConfig, { config }),
+  wrapProvider(WagmiProvider, { config, reconnectOnMount: false }),
```

Comment:
reconnectOnMount does not work for us because we can connect with a custom connector depending on which wallet we use.